WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dynamiccard/DynamicCardCallToActionButton.kt

```diff
@@ -0,0 +1,28 @@
+package org.wordpress.android.ui.mysite.cards.dynamiccard
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.domains.management.success
+
+@Composable
+fun DynamicCardCallToActionButton(
+    text: String,
+    onClicked: () -> Unit
+) {
+    TextButton(
+        modifier = Modifier.padding(start = 4.dp),
+        onClick = onClicked,
+    ) {
+        Text(
+            text = text,
+            style = MaterialTheme.typography.bodyMedium.copy(
```

Comment:
Would it make sense to reuse [the `DashboardCardTypography`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/ui/compose/styles/DashboardCardTypography.kt) like other dashboard cards (e.g. [the recent `BloganuaryNudgeCard`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloganuary/BloganuaryNudgeCard.kt)) or that would require falling back to `AppTheme` instead of the `M3Theme` 🤔

Reply:
All in all, I migrated the entire feature to AppTheme and used [DashboardCardTypography](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/ui/compose/styles/DashboardCardTypography.kt) to not mix up so many different approaches: Android View, M2 Compose, and M3 Compose.

Reply:
Makes sense 👍 
Thank you for the change Artyom 🙇